findphotodates.py

```
@@ -0,0 +1,318 @@
+        print(f"  {ext.upper()}: {count}")
+
+    # Calculate date summaries
+    date_counter = Counter()
+    for _, date, _ in photo_data:
+        if date:
+            try:
+                day = date.split(" ")[0]  # Extract just the date portion
+                date_counter[day] += 1
+            except (IndexError, AttributeError) as e:
+                # Skip invalid dates
+                if args.debug:
```

Ruff (0.8.2)

Undefined name `args`

(F821)